backend/signing/configuration.go

```diff
@@ -68,7 +71,17 @@ func NewSinglesigConfiguration(
 	extendedPublicKey *hdkeychain.ExtendedKey,
 ) *Configuration {
 	return NewConfiguration(
-		scriptType, absoluteKeypath, []*hdkeychain.ExtendedKey{extendedPublicKey}, 1)
+		scriptType, absoluteKeypath, []*hdkeychain.ExtendedKey{extendedPublicKey}, "", 1)
+}
+
+// NewAddressConfiguration creates a new account address configuration
+func NewAddressConfiguration(
+	scriptType ScriptType,
+	absoluteKeypath AbsoluteKeypath,
+	address string,
+) *Configuration {
+	return NewConfiguration(
+		scriptType, absoluteKeypath, []*hdkeychain.ExtendedKey{}, address, 1)
```

Comment:
maybe `0` instead (default value), as `signingThreshold` is not used in this configuration